pandas/core/generic.py

```diff
@@ -3263,30 +3263,54 @@ def to_latex(
             columns. By default, 'l' will be used for all columns except
             columns of numbers, which default to 'r'.
         longtable : bool, optional
-            By default, the value will be read from the pandas config
-            module. Use a longtable environment instead of tabular. Requires
+            Use a longtable environment instead of tabular. Requires
             adding a \usepackage{{longtable}} to your LaTeX preamble.
+            By default, the value will be read from the pandas config
+            module, and set to `True` if the option ``styler.latex.environment`` is
+            `"longtable"`.
+
+            .. versionchanged:: 2.0.0
+               The pandas option affecting this argument has changed.
```

Comment:
For these notes could we state what the new option is i.e. `...this argument has changed to  <new option>`

Reply:
The main part of the parameter doc string contains the names of the pandas option that are used. Not sure we need to explicitly state it again in the versionchanged only a couple lines below?

Did add the default values though.